napari/_qt/qt_main_window.py

```diff
@@ -311,7 +319,7 @@ def _help_changed(self, event):
         """
         self._help.setText(event.text)
 
-    def screenshot(self):
+    def screenshot(self, path=None):
```

Comment:
Can you add a doc string for `path` here (same one you used in `Viewer.py` is fine).